dry_adiabatic_adjust/dadadj.meta

```diff
@@ -126,9 +126,9 @@
   dimensions = ()
   intent = out
 [ errflg ]
-  standard_name = ccpp_error_flag
-  long_name = Error flag for error handling in CCPP
-  units = flag
+  standard_name = ccpp_error_code
+  long_name = Error code for error handling in CCPP
+  units = 1
```

Comment:
Just adding a second reminder to add an updated version of the `NamesNotInDictionary.txt` file to this PR following the instructions found online here:

https://github.com/ESCOMP/atmospheric_physics/wiki/Development-workflow#namesnotindictionarytxt

Reply:
Done.